tests/test_data/MAG_FORMAT_SAMPLE.MAG

```diff
@@ -0,0 +1,49 @@
+---- HEADER ----
+

```

Comment:
Are the empty lines part of the spec?

Reply:
Nope empty lines are ignored on read in but there's always empty lines added on writing.